rmf_human_detector_oakd/src/HumanDetector.cpp

```diff
@@ -146,6 +147,7 @@ HumanDetector::HumanDetector(
   stereo->depth.link(spatialDetectionNetwork->inputDepth);
   spatialDetectionNetwork->passthroughDepth.link(xoutDepth->input);
 
+// Function for the detection of the thread
```

Comment:
Not sure what you mean by "detection of the thread"? Did you mean
```suggestion
// Run detection in a separate thread
```